addons/delivery/models/res_partner.py

```diff
@@ -7,3 +7,4 @@ class ResPartner(models.Model):
     _inherit = 'res.partner'
 
     property_delivery_carrier_id = fields.Many2one('delivery.carrier', company_dependent=True, string="Delivery Method", help="Default delivery method used in sales orders.")
+    external_id = fields.Json(string="External Identifier", help="Information needed by shipping providers in case of pickup point addresses.")
```

Comment:
pickup_point_address no? is it really an id stored in it?

Reply:
It is different from a provider to another. In case of Sendcloud, it's just and id used by the provider later to process the shipment when the delivery is validated. In case of Fedex, it's an id in addition to other fields.